src/components/molecules/MessagePanel/MessagePanel.styled.tsx

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+
+import {Text} from '@custom-antd';
+
+import Colors from '@styles/Colors';
+
+export const MessagePanelWrapper = styled.div`
+  position: relative;
+
+  display: flex;
+  gap: 5px;
+
+  padding: 20px 50px 20px 20px;
+
+  &.error {
+    background: rgb(136 19 55 / 60%);
+    border: 1px solid ${Colors.rose500};
+    border-left: 0;
+    border-right: 0;
+  }
+
+  &.warning {
+    background: rgb(120 53 15 / 50%);
+    border: 1px solid ${Colors.amber400};
+    border-left: 0;
+    border-right: 0;
+  }
+
+  &.default {
+    background: ${Colors.indigo900};
+    border: 1px solid ${Colors.indigo600};
+    border-radius: 8px;
+  }
+
+  &.inline {
+    &.error {
+      border: 1px solid ${Colors.rose500};
+      border-radius: 4px;
+    }
+
+    &.warning {
+      border: 1px solid ${Colors.amber400};
+      border-radius: 4px;
+    }
+  }
+`;
+
+export const MessageDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: auto;
+  gap: 5px;
+`;
+
+export const MessageDescriptionText = styled(Text)`
+  a {
+    font-weight: 500;
+    text-decoration: underline;
+  }
+
+  &.error {
+    a {
+      color: ${Colors.rose200};
+    }
+  }
+
+  &.warning {
+    a {
+      color: ${Colors.amber200};
+    }
+  }
```

Comment:
Given it's the same color as for the description:
```suggestion
  a {
    font-weight: 500;
    text-decoration: underline;
    color: inherit;
  }
```

Reply:
Agree